src/date/format-properties.js

```diff
@@ -208,6 +209,56 @@ return function( pattern, cldr ) {
 
 			// Zone
 			case "z":
+
+				// z...zzz: "{shortRegion}", eg. "PST" or "PDT".
+				// zzzz: "{regionName} {Standard Time}" or "{regionName} {Daylight Time}",
+				// eg. "Pacific Standard Time" or "Pacific Daylight Time".
+				if ( properties.tzId ) {
+
+					// The latest metazone data of the metazone array.
+					//TODO expand to support the historic metazones based on the given date.
+					var metaZone = cldr.supplemental(
+						[ "metaZones/metazoneInfo/timezone/" + properties.tzId, 0 ]
+						);
+					if ( metaZone ) {
+						properties.exemplarCity = cldr.main(
```

Comment:
We do not need `exemplarCity` or `regionFormatStandard` and `regionFormatDaylight`. Let me change the code to get these data when `standardTzName` and `daylightTzName` are not found.